pkg/auth/providerrefresh/refresher.go

```diff
@@ -251,24 +253,9 @@ func (r *refresher) refreshAttributes(attribs *v3.UserAttribute) (*v3.UserAttrib
 
 		attribs.GroupPrincipals[providerName] = v32.Principals{Items: newGroupPrincipals}
 
-		if principalID != "" && len(loginTokens[providerName]) > 0 {
-			// A user is 1:1 with its principal for a given provider, no need to get principals from tokens beyond the first one
-			userPrincipal, err := providers.GetPrincipal(principalID, *loginTokens[providerName][0])
-			if err != nil {
-				return nil, err
-			}
-			userExtraInfo := providers.GetUserExtraAttributes(providerName, userPrincipal)
-			if userExtraInfo != nil {
-				if attribs.ExtraByProvider == nil {
-					attribs.ExtraByProvider = make(map[string]map[string][]string)
-				}
-				attribs.ExtraByProvider[providerName] = userExtraInfo
-			}
-		}
-
 		canAccessProvider := false
 
-		if principalID != "" {
+		if principalID != "" && !errorConfirmingLogins {
```

Comment:
Added to address an issue pointed out by @MbolotSuse which is that we're not 100% sure what the effect of calling `CanAccessWithGroupProviders` would be if we had run into an access error on line 229, so adding this should restore the previous behavior of skipping this section in such a case.